localstack-core/localstack/dev/kubernetes/__main__.py

```diff
@@ -17,7 +17,9 @@ def generate_k8s_cluster_config(pro: bool = False, mount_moto: bool = False, por
         }
     )
 
-    egg_path = os.path.join(root_path, "localstack_core.egg-info/entry_points.txt")
+    egg_path = os.path.join(
+        root_path, "localstack-core", "localstack_core.egg-info/entry_points.txt"
+    )
```

Comment:
q: Interesting, this seems to be a leftover from #10931. Which means the entrypoint mount wasn't working?

Reply:
Yep, noticed this coincidentally when checking the generated paths